jdisc_http_service/src/main/java/com/yahoo/jdisc/http/ssl/pem/package-info.java

```diff
@@ -0,0 +1,5 @@
+// Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+/**
+ * @author bjorncs
+ */
+package com.yahoo.jdisc.http.ssl.pem;
```

Comment:
Why a package-info.java without any annotations? Was the intentions to export this package?

Reply:
No, it is not supposed to be exported nor marked public API. It's leftover from the original code in `jdisc_http_filters`. I will remove the file.

Reply:
Fixed in new commit.